feat(integrations): Add troubleshooting docs for Vercel, JIRA and Azure DevOps (#3236)
getsentry/sentry-docs: src/docs/product/integrations/vercel/index.mdx

3. Delete the internal integration that was created by navigating to **Organization Settings** > **Developer Settings** and clicking the trash can icon next to **Vercel Internal Integration**. You will be prompted to confirm the deletion by entering a string provided in the modal. Enter the string and click "Confirm" to finalize the deletion.

   ![Sentry modal showing uninstalling Vercel integration](vercel_delete_internal_integration.png)

## Troubleshooting

### Failed to fetch

![](vercel_failed_to_fetch.png)

> This usually happens if you have an ad blocker that is blocking the conversation between Vercel and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.

Reviewer: ```suggestion
This issue typically occurs if you have an ad blocker blocking the conversation between Vercel and Sentry during setup. To remediate the issue, disable your ad blocker and go through the installation flow again.
```
